app/javascript/components/forms/validations.js

```diff
@@ -7,7 +7,7 @@ export const number = value =>
   (value && isNaN(Number(value)) ? 'Must be a number' : undefined);
 
 export const email = value =>
-  (value && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)
+  (value && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,10}$/i.test(value)
```

Comment:
What is the reason for the regex change?

Reply:
Allowing top level domains up to 10 chars long (.earth and longer)

Reply:
Muy bien